src/pages/my-courses/MyCourses.tsx

```diff
@@ -182,7 +138,7 @@ const MyCourses = () => {
     <>
       <MyCorsesCardsList
         deleteItem={onDelete}
-        duplicateItem={(itemId: string) => void handleDuplicate(itemId)}
+        duplicateItem={(itemId: string) => duplicateItem(itemId)}
```

Comment:
```suggestion
        duplicateItem={duplicateItem}
```

Take a look If u can do this ( you can check my CooperationNotes PR, I had the same)

Reply:
Yeah this works, thanks.